### ISSUE TITLE
Consensus Service Stop after Deploy Contract

### ISSUE THREAD

**Issue by cloud8little** [neo-project/neo#2123]
**Describe the bug**
this may related to https://github.com/neo-project/neo/pull/2119 

**To Reproduce**
1. Run a CN and Normal node.
2. Update NEP17.Owner.cs as the following:

```cs
        public static void Update(byte[] nefFile, byte[] manifest)
        {
            if (!IsOwner()) throw new Exception("No authorization.");
            Management.Update(nefFile, manifest);
            //Contract.Update(nefFile,manifest);
        }

        public static void Destroy()
        {
            if (!IsOwner()) throw new Exception("No authorization.");
            Management.Destroy();
            //Contract.Destroy();
        }
```
3. Add Management.cs to Neo.SmartContract.Framework

```cs
#pragma warning disable CS0626

using System.Numerics;
using Neo.SmartContract.Framework.Services.Neo;

namespace Neo.SmartContract.Framework.Services.Neo
{
    [Contract("0xdf18cb2476964c241558ed1e2e8881dcd2d50bde")]
    public class Management
    {
        public static extern void Update(byte[] nefFile, byte[] manifest);
        public static extern void Destroy();
    }
}
```
4. Deploy the contract, it is successful.
5. Then CN node relay blocks and then stop.

**Expected behavior**
CN node should proceed.

**Screenshots**
![bc22fc891cf4521c581a0558da0d140](https://user-images.githubusercontent.com/34291844/101343444-f7f67d00-38be-11eb-9463-a1a9962d0c63.png)
![e1d153ba78f05928ff3d21195a27ab0](https://user-images.githubusercontent.com/34291844/101343452-f927aa00-38be-11eb-97e9-816308425e76.png)

[testcontract.zip](https://github.com/neo-project/neo/files/5652238/testcontract.zip)

**Platform:**
 - OS: e.g. Windows 10 x64
 - Version neo #2119

**Comment by erikzhang** [comment]
This should be fixed by https://github.com/neo-project/neo/pull/2119/commits/695716aed1057b6f7b94d87f6b5610925f0b245a

**Comment by cloud8little** [comment]
> This should be fixed by [695716a](https://github.com/neo-project/neo/commit/695716aed1057b6f7b94d87f6b5610925f0b245a)

Tested OK with the commit 695716a
